File: DSA/DSA03015 MUA LƯƠNG THỰC.cpp
#include <bits/stdc++.h>
using namespace std;

void submit()
{
    int n, s, m;
    cin >> n >> s >> m;

    if (n < m || (n-m)*6 < m)
    {
        cout << -1;
    }
    else
    {
        cout << ceil((float)m*s/n);
    }


    cout << endl;
}

int main()
{
    ios::sync_with_stdio(false);                                    // Speed up input/output
    cin.tie(nullptr);
    cout.tie(nullptr);

    int test;
    cin >> test;
    while (test--)
    {
        submit();
    }
    

    return 0;
}

File: DSA_python/DSA03015 MUA LƯƠNG THỰC.py
import math

def submit():
    n, s, m = map(int, input().split())

    if n < m or (n-m)*6 < m:
        print(-1)
    else:
        print(math.ceil(m*s/n))

test = int(input())
for i in range(test):
    submit()